flink-table/flink-table-api-scala-bridge/src/main/java/org/apache/flink/table/operations/ScalaDataStreamQueryOperation.java

```diff
@@ -38,6 +39,7 @@
 @Internal
 public class ScalaDataStreamQueryOperation<E> implements QueryOperation {
 
+	private ObjectIdentifier identifier;
```

Comment:
Is it possible to have this during construction?

Reply:
We don't know the registered table name when construct this instance from `fromDataStream(DataStream<T> dataStream)`.

Reply:
So when the name is decided?